mixins/arrow-keys-mixin.md

```diff
@@ -51,4 +51,8 @@ customElements.define('my-element', MyElement);
 
 - `arrowKeysDirection` (String): Indicates which arrow keys are allowed (default is leftright)
 - `arrowKeysNoWrap` (Boolean): Whether focus should wrap from end-to-start and start-to-end
-- `arrowKeysBeforeFocus` (Function): Async callback invoked before focus us applied
+
+**Functions:**
+- `arrowKeysContainer`: Hooks up the event listener, to be used in your `render` function to wrap the focusable elements
+- `arrowKeysFocusablesProvider`: Override to provide the focusable elements if necessary (see above)
+- `arrowKeysOnBeforeFocus`: Override to provide an async callback invoked before focus is applied
```

Comment:
I might mention this is optional.  Background - I think tabs exclusively uses this to update some visuals (scroll position, expansion of collapsed tabs) before trying to apply focus, otherwise the browser will do it's own thing to scroll overflowing elements into view which messes up the designed behaviour.